aten/src/ATen/test/extension_backend_test.cpp

```diff
@@ -62,3 +66,42 @@ TEST(BackendExtensionTest, TestRegisterOp) {
         .aliasAnalysis(c10::AliasAnalysisKind::FROM_SCHEMA))
   );
 }
+
+// This is a "shitty" version of lazy tensor, where we run every operator by
+// running it on JIT
+struct ViaJitTensorImpl : public c10::TensorImpl {
```

Comment:
Hmmm I'm not sure whether it's necessary to write a hacky lazy tensor here. 
But I usually add my test in https://github.com/pytorch/pytorch/blob/master/test/cpp_extensions/msnpu_extension.cpp#L29. That's exactly how XLA is registered these days.

Reply:
This is my bad, the testing changes shouldn't be in this diff